xonsh/main.py

```diff
@@ -243,6 +243,10 @@ def main(argv=None):
             from xonsh import xonfig  # lazy import
             xonfig.xonfig_main(['wizard', '--confirm'])
         shell.cmdloop()
+    else:
+        print('xonsh: error: invalid argument {!r}'.format(sys.argv[1]))
```

Comment:
was thinking if i could decouple `premain` from error printing, will get back to this